README.md

```diff
@@ -47,6 +47,27 @@ import * as typeformEmbed from '@typeform/embed'
 const typeformEmbed = require('@typeform/embed')
 ```
 
+#### With React Hooks
+```jsx
+import React, { useRef, useEffect } from 'react';
+
+const MyTypeform = () => {
+  const typeformRef = useRef(null);
+  
+  useEffect(() => {
+    typeformEmbed.makeWidget(typeFormRef.current, 'https://form.typeform.com/to/MY_TYPEFORM_ID', {
```

Comment:
`typeFormRef is not defined` - you have a typo in the variable name (uppercase F).

```suggestion
    typeformEmbed.makeWidget(typeformRef.current, 'https://form.typeform.com/to/MY_TYPEFORM_ID', {
```